deployment/network-operator/values.yaml

```diff
@@ -32,7 +32,7 @@ sriovNetworkOperator:
 
 # Node Feature discovery chart related values
 node-feature-discovery:
-  enableNodeFeatureApi: false
+  enableNodeFeatureApi: true
```

Comment:
These are generated. Change in https://github.com/Mellanox/network-operator/blob/master/hack/templates/values/values.template

then run `make release-build` and commit the generated files